src/pages/black-friday.tsx

```diff
@@ -38,72 +54,109 @@ const BlackFriday = ({ lang, metatagsDescriptions, langJson, navbarLang, footerL
       description: langJson.blackFriday.FeatureSection.feature4.subtitle1,
     },
   ];
+
+  const onBusinessPlansSelected = (isBusiness) => {
+    setIsBusiness(isBusiness);
+  };
+
+  const onCheckoutButtonClicked = (priceId, isCheckoutForLifetime) => {
+    const lifetimeSpacePlan = products?.individuals[Interval.Lifetime].find((product) => product.priceId === priceId);
```

Comment:
Do you need that? Or you will use the same coupon code for all plans (Individuals | Lifetime | Business)